addons/mrp_repair/tests/test_flows.py

```diff
@@ -0,0 +1,50 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import Form, tagged
+from odoo.addons.mrp.tests.common import TestMrpCommon
+from odoo.exceptions import AccessError
+
+
+@tagged('post_install', '-at_install')
+class TestMrpRepairFlows(TestMrpCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env.ref('base.group_user').write({'implied_ids': [(4, cls.env.ref('stock.group_production_lot').id)]})
+
+    def test_possible_to_add_kit_after_confirm(self):
+        """
+        Test that it is possible to add a kit manufactured product to an already confirmed Repair Order
+        """
+        repaired = self.env['product.product'].create({'name': 'Repaired'})
+        part = self.env['product.product'].create({'name': 'Kit Component'})
+        kit = self.env['product.product'].create({
+            'name': 'Kit',
+            'type': 'product',
+        })
```

Comment:
Always better to work in batch 
```suggestion
        repaired, part, kit = self.env['product.product'].create([
            {...},
            ...
        ])
```